packaging/installer/netdata-uninstaller.sh

```diff
@@ -441,8 +492,29 @@ stop_all_netdata() {
 
 trap quit_msg EXIT
 
+detect_existing_install() {
+  # shellcheck disable=SC2154
+  ndpath="$(PATH="${searchpath}" command -v netdata 2>/dev/null)"
+
+  if [ -z "$ndpath" ] && [ -x /opt/netdata/bin/netdata ]; then
+    ndpath="/opt/netdata/bin/netdata"
+  fi
+
+  if [ -n "${ndpath}" ]; then
+    ndprefix="$(dirname "$(dirname "${ndpath}")")"
+  fi
+
+  if echo "${ndprefix}" | grep -Eq '/usr$'; then
+    ndprefix="$(dirname "${ndprefix}")"
+  fi
+
+  INSTALL_PREFIX="${ndprefix}"
+}
+
 #shellcheck source=/dev/null
-source "${ENVIRONMENT_FILE}" || exit 1
+detect_existing_install
+# shellcheck disable=SC1090
+. "${INSTALL_PREFIX}${ENVIRONMENT_FILE}" || exit 1
```

Comment:
We should probably support passing the install prefix directly from the kickstart script instead of always searching for it here.